public/data/khushi61.json

```
@@ -0,0 +1,23 @@
+{
+  "name": "Khushi Shukla",
+  "type": "personal",
+  "bio": "Learner| FrontendDev ^__^ ",
+  "avatar": "https://avatars.githubusercontent.com/u/81404366?s=400&u=02963fadebb7a1bef546afc02189d9dccf624f77&v=4",
```

This looks a lot better and short! 

```suggestion
  "avatar": "https://github.com/khushi818.png",
```